crates/sol-macro-gen/src/sol_macro_gen.rs

```diff
@@ -336,7 +336,7 @@ edition = "2021"
                 version
             )
         } else {
-            &r#"alloy = {{ git = "https://github.com/alloy-rs/alloy", features = ["sol-types", "contract"] }}"#.to_string()
+            &r#"alloy = { git = "https://github.com/alloy-rs/alloy", features = ["sol-types", "contract"] }"#.to_string()
```

Comment:
pls also do this above
```suggestion
            r#"alloy = { git = "https://github.com/alloy-rs/alloy", features = ["sol-types", "contract"] }"#.to_string()
```